docs/configuration/index.md

```diff
@@ -961,9 +962,29 @@ Issuing a GET request at the same URL will return the spec that is currently in
+|`maxNonPrimaryReplicantsToLoad`|The maximum number of replicas that can be assigned across all tiers in a single Coordinator run. This parameter serves the same purpose as `replicationThrottleLimit` except this limit applies at the cluster-level instead of per tier. The default value essentially means that there is no limit on the number of replicas assigned per coordination cycle. If you want to use a non-default value for this config, you may want to start with it being `~20%` of the number of segments found on the Historical server with the most segments. Use the Druid metric, `coordinator/time` with the filter `duty=org.apache.druid.server.coordinator.duty.RunRules` to see how different values of this config impact your Coordinator execution time.|`Integer.MAX_VALUE` (i.e. no limit)|
 
+##### Smart segment loading
 
+The `smartSegmentLoading` mode of the Coordinator makes configuring it for segment loading and balancing much easier.
+In this mode, the Coordinator does not require the user to provide values of the following parameters and computes them automatically instead.
+**If provided, the values are simply ignored.**
+The computed values are based on the current state of the cluster and are meant to optimize the performance of the Coordinator.
+
+|Property|Computed value|Explanation|
+|--------|--------------|-----------|
+|`useRoundRobinSegmentAssignment`|true|Speeds up segment assignment|
+|`maxSegmentsInNodeLoadingQueue`|0|Removes the limit on load queue size|
+|`replicationThrottleLimit`|2% of used segments, minimum value = 100|Ensures that replication is not done too aggressively in case of a historical disappearing only intermittently.|
+|`replicantLifetime`|60|Allows segments to wait about an hour (assuming a coordinator period of 1 minute) in the load queue before an alert is raised. This value is higher than the previous default of 15 because in `smartSegmentLoading` mode, load queues are not limited by size. Thus, segments might get assigned to a load queue even if the corresponding server is slow to load them.|
+|`maxNonPrimaryReplicantsToLoad`|`Integer.MAX_VALUE` (no limit)|This throttling is already handled by `replicationThrottleLimit`.|
+|`maxSegmentsToMove`|2% of used segments, minimum value = 100, maximum value = 1000|Ensures that some segments are always moving in the cluster to keep it well balanced. The maximum value keeps the coordinator run times bounded.|
+|`decommissioningMaxPercentOfMaxSegmentsToMove`|100|Prioritizes move of segments from decommissioning servers so that they can be terminated quickly.|
```

Comment:
```suggestion
|`decommissioningMaxPercentOfMaxSegmentsToMove`|100|Prioritizes the move of segments from decommissioning servers so that they can be terminated quickly.|
```